target/appengine-staging/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this file for? Why do you need multiple appengine-web.xml files?

Reply:
We need to add gitignore for the /target directory, so that those files can be ignored

Reply:
The appengine-web.xml is supposed to be a configuration file. However, I realize now that it shouldn't be in this location. I removed all the extra appengine-web.xml files and now I have the one in the src folder and another one in the target/WEB-INF/ folder (this one si generated by the engine when it runs).